src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -431,8 +431,35 @@ def tune_for_tgi_jumpstart(self, max_tuning_duration: int = 1800):
             sharded_supported=sharded_supported, max_tuning_duration=max_tuning_duration
         )
 
+    def set_deployment_config(self, config_name: Optional[str]) -> None:
+        """Sets the deployment config to apply to the model.
+
+        Args:
+            config_name (Optional[str]):
+                The name of the deployment config. Set to None to unset
+                any existing config that is applied to the model.
+        """
+        if self.pysdk_model is None:
```

Comment:
I don't think this is necessary. Customers won't call set before list/display. You can remove this in future iterations.

Reply:
Should an exception be thrown instead? if they try to call this before list/display?